references/proxy-template/generate-proxy.sh

```diff
@@ -38,11 +71,19 @@ sed -i.bak "s|@Proxy@|$PROXY|" ./"$PROXY"-"$VERSION"/test/features/step_definiti
 sed -i.bak "s|@Version@|$VERSION|" ./"$PROXY"-"$VERSION"/apiproxy/proxies/default.xml
 sed -i.bak "s|@Version@|$VERSION|" ./"$PROXY"-"$VERSION"/package.json
 sed -i.bak "s|@Version@|$VERSION|" ./"$PROXY"-"$VERSION"/test/features/step_definitions/init.js
-sed -i.bak "s|@TargetURL@|$TARGETURL|" ./"$PROXY"-"$VERSION"/apiproxy/targets/default.xml
+sed -i.bak "s|@ProxyPath@|$PROXY_PATH|" ./"$PROXY"-"$VERSION"/apiproxy/proxies/default.xml
+sed -i.bak "s|@ProxyPath@|$PROXY_PATH|" ./"$PROXY"-"$VERSION"/test/features/TargetServer.feature
+sed -i.bak "s|@TargetServerName@|$TARGET_SERVER_NAME|" ./"$PROXY"-"$VERSION"/apiproxy/targets/default.xml
 
 rm ./"$PROXY"-"$VERSION"/apiproxy/proxies/default.xml.bak
 rm ./"$PROXY"-"$VERSION"/package.json.bak
 rm ./"$PROXY"-"$VERSION"/test/features/step_definitions/init.js.bak
+rm ./"$PROXY"-"$VERSION"/test/features/TargetServer.feature.bak
 rm ./"$PROXY"-"$VERSION"/apiproxy/targets/default.xml.bak
 
+export APIGEETOOL_DEPLOY_TARGETSERVER_SSLOPTION
+export TARGET_HOST
+export TARGET_PORT
+export TARGET_SERVER_NAME
```

Comment:
@JoelGauci  What's the purpose of exporting these here?

Reply:
@danistrebel : I need these variables to test if target server exists and to create it if it does not (using **npm run deployTargetServer**)